### PR TITLE
Try to implement custom darken lighten value input

### LINKED ISSUE #9: Granular lighten/darken
Darken/Lighten with hard-coded `10%` is not going to be enough for everyone.

Ideally having an input that you can put any value: `0.333333` or `50`.

---

I mean, shortcuts are great and should stay:

- Darken (10%)
- Lighten (10%)
- Darken (Custom value)
- Lighten (Custom value)

Where picking custom value would bring another Quick Pick inviting to enter the number value.

### PR DESCRIPTION
**Pull request by usernamehw**
Fixes #9

Is it intentional that label says `10%` but the actual value is `5`?

### CODE REVIEW

**Comment by ryanraposo** [comment]
So the brightness methods are probably the least set in stone. I found the darken/lighten from tinycolor to give unexpected results, do you feel the same? I see two options, either move away from tinycolor as a dependency, or list the adjustments differently- maybe just Brightness (-2), Brightness (-1) Brightness(+1) Brightness (+2) etc... in that solution even with an unreliable implementation there is less guarantee/expectation than something as strict as percentage, and it consolidates the commands and reduces the number of quickpicks.

For now though, your pull request looks great and if you could just add a placeholder or something to indicate whats expected on entry (eg percentage instead of number) i'll merge it. Awesome job with that, if there are any blocks moving forward that are hard to follow or anything like that just message me. Documenting methods is in the plans but until then just ask anytime

**Comment by usernamehw** [comment]
Changed the prompt to containt `Percent` word. 

Also, the input now will accept string containing `%` sign.
